### ISSUE TITLE
Does it supports input messages?

### ISSUE THREAD

**Issue by Barabba11** [andrei-tatar/node-red-contrib-nora#76]
Actually the status of my devices can be change by Homekit, Alexa, switches on the wall, I need then to update the status of lights in Google home too to reflect the changes, is it possible? Exactly for me is important on/off and brightness value.
Thank you for your kind support

**Comment by andrei-tatar** [comment]
Yes it is. Just send a message to the node similar to the output message of that node and the state will be updated and reported to google.

**Comment by Barabba11** [comment]
Thank you for your kind support. Well yes, it is updated value, but only if I open the light (click on the icon, on the main Google home page) and the "wheel" with the switch in the center appear. 
I explain better, on the Google Home main page there is the icon with the symbol of light, I think it is a static icon, it doesn't show the status and brighness, and the name is generic "light". I was expecting to see the same name of the node I placed on the flow, is it normal behaviour?
Maybe google home isn't good app? Maybe there are other apps which takes infos from google home and show name, status, and there will be more easier to change values without open the wheel? Thank you a lot!

**Comment by qeggs** [comment]
I am not able to get any response after injecting message into a NORA dimmable light node. The indicated state on Google Home does not change, and the light does not turn ON as it should.

To turn the light ON, I believe I should inject this JSON as the msg.payload:
`{"online":true,"on":true,"brightness":100}`

![image](https://user-images.githubusercontent.com/52592447/94195274-cace2d00-fe80-11ea-8f3d-c3cc7387f082.png)

**Comment by Barabba11** [comment]
> 
> 
> I am not able to get any response after injecting message into a NORA dimmable light node. The indicated state on Google Home does not change, and the light does not turn ON as it should.
> 
> To turn the light ON, I believe I should inject this JSON as the msg.payload:
> `{"online":true,"on":true,"brightness":100}`
> 
> ![image](https://user-images.githubusercontent.com/52592447/94195274-cace2d00-fe80-11ea-8f3d-c3cc7387f082.png)

you're right, that message, but the reaction will be present only in the main google home panel (where are all devices), and not on the sigle lamp.. for some reasons Google never thought to impement that (as he's not waiting changes in that moment), pity.
It works, the topic isn't important
